sklearn/tree/tests/test_tree.py

```diff
@@ -2549,7 +2549,8 @@ def test_missing_values_poisson():
         (datasets.make_classification, DecisionTreeClassifier),
     ],
 )
-def test_missing_values_is_resilience(make_data, Tree):
+@pytest.mark.parametrize("sample_weight_train", [None, "ones"])
```

Comment:
Should we also test the behavior when using non-uniform weights?

Reply:
Probably, an easier test than non-uniform weight is to assign 0-weight to some specific samples.

Reply:
It might be worth having a separated test for checking an equivalence.